source/configure/config-proxy-apache2.rst

```diff
@@ -42,16 +99,15 @@ On a Debian-based operating system such as Ubuntu, Apache2 proxy configuration i
             Require all granted
             ProxyPass http://127.0.0.1:8065/
             ProxyPassReverse http://127.0.0.1:8065/
-            ProxyPassReverseCookieDomain 127.0.0.1 mysubdomain.mydomain.com
+            ProxyPassReverseCookieDomain 127.0.0.1 mymattermost.tld
           </Location>
 
+          SSLEngine On
+          SSLCertificateFile /path/to/your/certificate.pem
+          SSLCertificateKeyFile /path/to/your/certificate.key
+          
+          # Uncomment the following line if your Mattermost server is requiring SSL/TLS connections
+          # SSLProxyEngine On
         </VirtualHost>
 
-5. (Debian/Ubuntu only) Because you'll likely have not set up the subdomain before now on Apache2, run ``a2ensite mysubdomain.mydomain.com`` to enable the site (do not run ``a2ensite mysubdomain.mydomain.com.conf``).
-
-6. Restart Apache2.
-
-    - On Ubuntu 14.04 and RHEL 6: ``sudo service apache2 restart``
-    - On Ubuntu 16.04+ and RHEL 7+: ``sudo systemctl restart apache2``
-
-You should be all set! Ensure that your Mattermost config file is pointing to the correct URL (which may include a port), and then ensure that your socket connection is not dropping once deployed. To prevent external access to Mattermost on port 8065, in the config file, set ``ListenAddress`` to ``localhost:8065`` instead of ``:8065``.
+ 3. If you have enabled TLS in the Mattermost settings, you must also enable the SSL module in Apache (usually with ``a2enmod ssl``) and change the ``http://`` part of the proxy destination to ``https://`` and the websocket protocol from ``ws://`` to ``wss://`` additionally to enabling the ``SSLProxyEngine``.
```

Comment:
@anx-ag - I suspect that the single blank space immediately before step 3 is what's causing the step text to display as part of the code block:

![Screenshot 2023-03-03 at 10 59 03 AM](https://user-images.githubusercontent.com/74422101/222768176-e7a22941-e70c-43b3-96da-5176a4e80310.png)

Reply:
Oh... Sorry for that :-( Should I resubmit or can you fix that quickly?

Reply:
If you're open to addressing the issues I've identified in this PR as a batch, feel free to do so. Let me know if you'd like my assistance with these formatting issues, and I can submit changes.

Reply:
Thanks, removing the space fixed it.